src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/ConfigurationManager.cs

```diff
@@ -243,6 +252,15 @@ private static void PreloadConfigurationSectionGroup(ConfigurationSectionGroup s
                 PreloadConfigurationSectionGroup(childSectionGroup);
         }
 
+        private static bool s_disableConfigurationManager;
+        private static bool DisableConfigurationManager
+        {
+            get
+            {
+                return AppContext.TryGetSwitch(@"Switch.System.Configuration.ConfigurationManager.DisableConfigurationManager", out s_disableConfigurationManager) ? s_disableConfigurationManager : false;
```

Comment:
```suggestion
                return AppContext.TryGetSwitch(@"System.Configuration.ConfigurationManager.DisableConfigurationManager", out s_disableConfigurationManager) ? s_disableConfigurationManager : false;
```

We don't use the `Switch.` prefix for feature switches. See https://github.com/dotnet/runtime/blob/main/docs/workflow/trimming/feature-switches.md